.tekton/kn-plugin-func-func-util-115-pull-request.yaml

```diff
@@ -54,7 +54,8 @@ spec:
               memory: 4Gi
   pipelineRef:
     name: docker-build
-  taskRunTemplate: {}
+  taskRunTemplate:
+    serviceAccountName: build-pipeline-kn-plugin-func-func-util-115
```

Comment:
this is the konflux build pipeline service acc migration, as disscussed in internal redhat slack [here](https://redhat-internal.slack.com/archives/CD87JDUB0/p1745564149845979)
Closing this since we manage our konflux manifest from elsewhere